src/App.tsx

```diff
@@ -1,9 +1,19 @@
-import React from 'react';
+import React, { useState } from 'react';
 import './App.scss';
 import { MoviesList } from './components/MoviesList';
 import moviesFromServer from './api/movies.json';
 
 export const App: React.FC = () => {
+  const [query, setQuery] = useState('');
+
+  const visibleMovies = [...moviesFromServer].filter(movie => {
+    const title = movie.title.toLowerCase().includes(query.toLowerCase());
```

Comment:
`query.toLowerCase()` - DRY. It's repeatable expression. Move to variable and reuse